octokit/search.go

```diff
@@ -2,47 +2,60 @@ package octokit
 
 import (
 	"github.com/jingweno/go-sawyer/hypermedia"
-	"net/url"
 )
 
-var SearchURL = Hyperlink("search{/type}?q={query}{&page,per_page,sort,order}")
+var SearchURITemplate = "search{/type}?q={query}{&page,per_page,sort,order}"
```

Comment:
I like keeping this a [`Hyperlink`][link] for convenience.

[link]: https://github.com/octokit/go-octokit/blob/master/octokit/hyperlink.go